### PR TITLE
Stop building ARM for ProjectReunion Foundation

### PR DESCRIPTION
**Pull request by kythant**
There was a decision to not support ARM and some of the packages we are using will not have an ARM version. 

Don't want to do have to do something like
#ifndef arm32
in our code.

### CODE REVIEW

**Comment by huichen123** [comment]
please search "buildPlatform". there are other places where arm is built

File: build/ProjectReunion-BuildFoundation.yml [hunk 1]
```diff
@@ -29,10 +29,6 @@ jobs:
         buildConfiguration: 'Release'
         normalizedConfiguration: 'fre'
         PGOBuildMode: 'Optimize'
-      Release_Arm:
-        buildPlatform: 'arm'
-        buildConfiguration: 'Release'
-        normalizedConfiguration: 'fre'
       Release_Arm64:
```
Comment (DrusTheAxe) [inline build/ProjectReunion-BuildFoundation.yml:32]: Why do Release_x86 + Release_x64 have PGOBuildMode:Optimize and Release_Arm64 doesn't?

**Review by DrusTheAxe — APPROVED** [review]

**Review by BenJKuhn — APPROVED** [review]

**Review by EHO-makai — APPROVED** [review]
I assume we will follow up with the other places that attempt to build or package up arm32.

File: build/ProjectReunion-BuildFoundation.yml (hunk 1 above)
Comment (kythant) [reply build/ProjectReunion-BuildFoundation.yml:32]: Sounds like optimize is good to have for a release build. Should I add it to Arm64?

**Comment by huichen123** [comment]
please also search "arm".

File: build/ProjectReunion-BuildFoundation.yml (hunk 1 above)
Comment (BenJKuhn) [reply build/ProjectReunion-BuildFoundation.yml:32]: Good question to save as an issue / bug. I wouldn't get hung up on it in this review.

**Review by huichen123 — APPROVED** [review]

**Pull request merged**